libs/components/FaucetAccountInput.tsx

```diff
@@ -13,16 +14,15 @@ const FaucetAccountInput: FC<FaucetAccountInputProps> = ({
 }) => {
 	return (
 		<div css={styles.addressInputContainer}>
-			<AccountIdenticon
-				css={styles.accountIdenticon}
-				theme="beachball"
-				size={28}
-				value={address ? address : PLACEHOLDER_ADDRESS}
-			/>
+			{address ? (
+				<AccountIdenticon theme="beachball" size={28} value={address} />
+			) : (
+				<Image src={CENNZnetBlue} width={28} height={28} alt="cennz-logo" />
+			)}
```

Comment:
Avoid ternary if possible, this can be updated to be

```
{!!address && <AccountIdenticon theme="beachball" size={28} value={address} />}
{!address && <Image src={CENNZnetBlue} width={28} height={28} alt="cennz-logo" />}
```